### ISSUE TITLE
Windows seems to be empty, but data isn't shown

### ISSUE THREAD

**Issue by aDresl** [hugolabe/Wike#172]
![Bildschirmfoto_wike](https://github.com/hugolabe/Wike/assets/67395155/347d7631-a814-4980-97ad-d643bee9f693)

Like here: [https://github.com/hugolabe/Wike/issues/136](url) the windows is empty. I followed all the steps there, but without success. Maybe the problem is different and needs to be solved separately.

As you see above, there is some data, but it's invisible. You can mark it blindly an drag it around, but if you unleash the mouse button, it disappears. How can I solve this?

When run from command line, I get this:

flatpak run com.github.hugolabe.Wike
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Keine Berechtigung
Failed to create GBM buffer of size 1185x944: Keine Berechtigung
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Keine Berechtigung
Failed to create GBM buffer of size 1185x944: Keine Berechtigung
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Keine Berechtigung
Failed to create GBM buffer of size 1185x944: Keine Berechtigung
Failed to create EGL images for DMABufs with file descriptors -1, -1 and -1

"Keine Berechtigung" is: no permission 

some additional informations:

inxi -S
Host: *****PC Kernel: 5.15.0-91-generic x86_64 bits: 64
    Desktop: Cinnamon 5.6.8 Distro: Linux Mint 21.1 Vera

flatpak --version
Flatpak 1.12.7

dpkg -s libglib2.0-bin | grep '^Version:'
Version: 2.72.4-0ubuntu2.2

flatpak run --command=ls com.github.hugolabe.Wike -la /app/share/runtime/locale
insgesamt 16
drwxr-xr-x 4 nfsnobody nfsnobody 4096 15. Dez 08:51 .
drwxr-xr-x 3 nfsnobody nfsnobody 4096  1. Jan 1970  ..
drwxr-xr-x 3 nfsnobody nfsnobody 4096  1. Jan 1970  de
drwxr-xr-x 3 nfsnobody nfsnobody 4096  1. Jan 1970  fr
-rw-r--r-- 1 nfsnobody nfsnobody    0 15. Dez 08:51 .ref

**Comment by hugolabe** [comment]
From what I've seen from a quick search, these error messages seem to be related to Webkit, but frankly I have no idea what can be done in this specific case.

Let's see if someone with more knowledge about these topics can provide some ideas.

**Comment by camelCaseNick** [comment]
Does this also happen with GNOME Web, NewsFlash, Apostrophe or Geary or another GNOME app using WebKit for you? If you have any, and do/don't experience this issue there, how did you install them? Do you by any chance have an Nvidia graphics card? If so, which drivers do you have, and how did you install them?

You could try running it with a newer WebKit2GTK version. Currently, Wike still ships with GNOME 44, but you can find (nearly) Wike 2.1.0 with the GNOME 45 runtime at https://github.com/hugolabe/Wike/actions/runs/7105263343.

If that still has the issue, on GNOME 45 and later, you can try `WEBKIT_DISABLE_DMABUF_RENDERER=1`.

**Comment by TobberH** [comment]
I have just installed Wike and had the exact same issue. The article window is just renders blank even though the data seems to be there. I could fix the issue by:

1. Install Flatseal.
2. Open Flatseal and select the Wike app.
3. Scroll to Environment and add `WEBKIT_DISABLE_DMABUF_RENDERER=1` as suggested by @camelCaseNick

**Comment by camelCaseNick** [comment]
@TobberH Does this also happen with other apps that use web views, like Apostrophe, NewsFlash, or Web? And do you get the same messages when you run `flatpak run com.github.hugolabe.Wike` (without `WEBKIT_DISABLE_DMABUF_RENDERER=1` set) as the original report?

Wike most likely cannot do anything here, so it would be best reported to WebKitGTK: https://bugs.webkit.org/.

**Comment by TobberH** [comment]
@camelCaseNick yeah, it does seem to be a bug with WebKitGTK. I haven't tried the other apps yet, but I can give it a go and see if they have same problem.

**Comment by TobberH** [comment]
@camelCaseNick I get these error messages when running without `WEBKIT_DISABLE_DMABUF_RENDERER=1`

```
Failed to create /home/REDACTED/.var/app/com.github.hugolabe.Wike/cache for shader cache (No such file or directory)---disabling.
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1669x1042: Permission denied
```

**Comment by TobberH** [comment]
Oh and also relevant: I get this error message both with AND without `WEBKIT_DISABLE_DMABUF_RENDERER=1`

**Comment by TobberH** [comment]
@camelCaseNick tried the NewsFlash app and it has same problem and give exactly the same error messages.
